code/game/turfs/change_turf.dm

```diff
@@ -135,7 +135,7 @@ GLOBAL_LIST_INIT(blacklisted_automated_baseturfs, typecacheof(list(
 
 	return W
 
-/turf/open/ChangeTurf(path, list/new_baseturfs, flags = CHANGETURF_INHERIT_AIR)
+/turf/open/ChangeTurf(path, list/new_baseturfs, flags)
```

Comment:
```suggestion
+ /turf/open/ChangeTurf(path, list/new_baseturfs, flags) //Resist the temptation to make this default to keeping air.
```

Reply:
About this. I can add CHANGETURF_INHERIT_AIR flag to all ChangeTurf, where we need keeping old air, and result be same. CHANGETURF_INHERIT_AIR mechanicks just need some rework.